packages/playwright/src/reporter.ts

```diff
@@ -340,22 +350,37 @@ export default class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
-        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-browser");
 
-        const output: string[] = [];
+      const output: string[] = [];
+
+      const executedProjectWithReplay = !Object.keys(this._projects).some(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && usingReplay;
+      });
+
+      if (!executedProjectWithReplay) {
+        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-project");
         output.push(emphasize("None of the configured projects ran using Replay Chromium."));
-        if (!existsSync(getRuntimePath())) {
+      }
+
+      if (executedProjectWithReplay && !existsSync(getRuntimePath())) {
```

Comment:
Would it make more sense for this check to be...
```suggestion
      if (output.length || (executedProjectWithReplay && !existsSync(getRuntimePath()))) {
```

Reply:
No that's not quite right either.